projects/packages/stats-admin/src/class-rest-controller.php

```diff
@@ -173,19 +211,26 @@ public function get_site_sub_resource_from_wpcom( $req ) {
 	 * @return array
 	 */
 	public function get_stats_single_resource_from_wpcom( $req ) {
-		return static::request_as_blog_cached(
-			sprintf(
-				'/sites/%d/stats/%s/%d?%s',
-				Jetpack_Options::get_option( 'id' ),
-				$req->get_param( 'resource' ),
-				$req->get_param( 'resource_id' ),
-				http_build_query(
-					$req->get_params()
-				)
-			),
-			'1.1',
-			array( 'timeout' => 30 )
-		);
+		switch ( $req->get_param( 'resource' ) ) {
+			case 'post':
+				return $this->wpcom_stats->get_post_views(
+					intval( $req->get_param( 'resource_id' ) ),
+					http_build_query(
```

Comment:
You don't need the `http_build_query` here as `get_post_views` accepts and array of query params :) 
Same goes for all methods in `WPCOM_Stats`